index.html

```diff
@@ -152,8 +152,9 @@
           </div>
 
           <div class="bd-toc-item">
-            <a class="bd-toc-link" href="#options/table-ajax.html">Options</a>
```

Comment:
Are this change removing the table-ajax.html?

Reply:
No, i just changed the link from our `options` menu entry to the new first option link (accent neutralise).

Reply:
Check the next change (Line 157) :-)